finalProject.html

```diff
@@ -0,0 +1,39 @@
+<html>
```

Comment:
please delete the index.html if you want this to be the main html file of your project